_posts/2020-01-01-2019-summary.md

```diff
@@ -0,0 +1,178 @@
+  and [`from_encoded_polyline`](/docs/current/functions/geospatial.html#from_encoded_polyline) functions.
+* [`geometry_from_hadoop_shape`](/docs/current/functions/geospatial.html#geometry_from_hadoop_shape) to decode geometry objects in 
+  Spatial Framework for Hadoop representation
+
+## Security
+
+* [Role-based access control](/docs/current/sql/create-role.html) and related commands.
+* [INVOKER security mode](/docs/current/sql/create-view.html#security) for views, which allows views to be run using the permissions of the 
+  current user.
+* Prevent replay attacks and result hijacking in client APIs.
+* JWT-based [internal communication](/docs/current/security/internal-communication.html#internal-authentication) authentication.
+* Credential passthrough, which allows Presto to authenticate with the underlying data source with 
+  credentials provided by the user running a query.
+* Impersonation for [Hive metastore](/docs/current/connector/hive.html#hive-thrift-metastore-configuration-properties).
+* Support for reading and writing encrypted files in HDFS using Hadoop KMS.
+
+## Cloud Integration
+
+* Support for Azure Data Lake Blob and ADLS Gen2 storage.
+* Support for [Google Cloud Storage](/docs/current/connector/hive-gcs-tutorial.html).
+* Several [performance improvements](https://github.com/prestosql/presto/pull/142) for AWS S3.
+
+## CLI
+
+* Syntax highlighting and multi-line editing.
+ 
+  ![](/assets/blog/2019-review/presto-cli.jpeg)
+
+* JSON output format and improvements to CSV output format.
+
+## New Connectors
+
+* [Elasticsearch](/docs/current/connector/elasticsearch.html)
+* [Google Sheets](/docs/current/connector/googlesheets.html)
+* [Amazon Kinesis](/docs/current/connector/kinesis.html)
+* [Apache Phoenix]({% post_url 2019-06-04-phoenix-connector %})
+* [MemSQL](/docs/current/connector/memsql.html)
+* Apache Iceberg
+
+## Other Improvements
+
+* [Docker image](https://hub.docker.com/r/prestosql/presto) that provides an out-of-the-box single node 
+  cluster with the JMX, memory, TPC-DS, and TPC-H catalogs. It can be deployed as a full cluster by 
+  mounting in configuration.
+* LZ4/ZSTD support for Parquet and ORC. These are the recommended compression algorithms.
+* Support for insert-only Hive transactional tables and bucketing v2 as part of 
+  [making Presto compatible with Hive 3]({% post_url 2019-12-28-hive-3 %}).
+* Improvements in ANALYZE statement for Hive connector.
+* Support for [multiple files per bucket]({% post_url 2019-05-29-improved-hive-bucketing %}) 
+  for Hive tables. This allows inserting data into bucketed tables without having to rewrite entire partitions.
+* Lots and lots of bug fixes!
+
+# Coming Up...
+
+These are some of the projects that are currently in progress and are likely to land in the short term.
+
+* Support for pushing down row dereference expressions into connectors. This will help reduce 
```

Comment:
you might mention
* positions pushdown to reader (https://github.com/prestosql/presto/pull/2254)
* extended late materializaton support for complex subqueries (https://github.com/prestosql/presto/pull/2418)
* Improved insert to partitioned tables (https://github.com/prestosql/presto/pull/2358)